components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -1,25 +1,237 @@
 #include "ekat/ekat_assert.hpp"
 #include "physics/rrtmgp/scream_rrtmgp_interface.hpp"
 #include "physics/rrtmgp/atmosphere_radiation.hpp"
+#include "physics/rrtmgp/rrtmgp_inputs_initializer.hpp"
+#include "mo_gas_concentrations.h"
 
 namespace scream {
     RRTMGPRadiation::RRTMGPRadiation (const ekat::Comm& comm, const ekat::ParameterList& params) : m_rrtmgp_comm (comm), m_rrtmgp_params (params) {
```

Comment:
It might be good to include the constructor for the `AtmosphereProcess` base class at the beginning of this initializer list, to make clear that this is how construction works (and to have an obvious place to change something if that base class constructor is altered).

Reply:
I'm confused; do you mean to call the base class constructor at the top of the RRTMGPRadiation constructor?

Reply:
Like this?
```
    RRTMGPRadiation::RRTMGPRadiation (const ekat::Comm& comm, const ekat::ParameterList& params)                                                                    
        : AtmosphereProcess::AtmosphereProcess(), m_rrtmgp_comm (comm), m_rrtmgp_params (params) {
        /*
         * Anything that can be initialized without grid information can be initialized here.
         * I.e., universal constants, options, etc.
         */
          if (!m_rrtmgp_params.isParameter("Grid")) {
              m_rrtmgp_params.set("Grid",std::string("SE Physics"));
          }

          m_initializer = create_field_initializer<RRTMGPInputsInitializer>();
    }  // RRTMGPRadiation::RRTMGPRadiation
```

Reply:
Yup, exactly.

Reply:
Done. Thanks!